lib/Hakyll/Web/Template/Context.hs

```diff
@@ -288,7 +299,16 @@ metadataField = Context $ \k _ i -> do
 
 
 --------------------------------------------------------------------------------
--- | Absolute url to the resulting item
+-- | Absolute url to the resulting item.  For an example
+-- underlying file @posts/foo.markdown@ and result file @posts/foo.html@,
+-- the context
+--
+-- > urlField "url"
+--
+-- is equivalent to
+--
+-- > constField "url" "posts/foo.html"
+--
```

Comment:
I find this example a bit contrived. Can't it say that "for an item that produces a file _posts/foo.html_, this field contains "posts/foo.html"?

Ditto for `pathField` and `titleField`; I feel those examples would be more readable as prose, not code.

Reply:
I switched the examples to prose.  What do you think?

Reply:
I think it's perfect now, thank you!